server/tserver/src/main/java/org/apache/accumulo/tserver/ScanServer.java

```diff
@@ -896,6 +918,10 @@ public InitialMultiScan startMultiScan(TInfo tinfo, TCredentials credentials,
           ssio, authorizations, waitForWrites, tSamplerConfig, batchTimeOut, contextArg,
           executionHints, getBatchScanTabletResolver(tablets), busyTimeout);
 
+      if (!reservation.getFailures().isEmpty()) {
+        ims.result.setFailures(reservation.getFailures());
+      }
+
```

Comment:
I was wondering if we need to handle the case of the delegate having already set some failures.   Digging around to answer that question I found [this code](https://github.com/apache/accumulo/blob/af6b4978d42e7a1f9102a7ec9605be51c67bb595/server/tserver/src/main/java/org/apache/accumulo/tserver/scan/LookupTask.java#L108-L113).  Looking at that I think as long the tablet resolver does not return anything that failed then it will naturally be added to the failure by delegate.startMultiScan().  I added comments elsewhere about getting the tablet resolver to not contain failures.  

```suggestion
```

Reply:
Addressed in 381dac3